packages/flutter/test/material/dialog_test.dart

```diff
@@ -58,15 +62,29 @@ void main() {
     await tester.pumpWidget(_appWithAlertDialog(tester, dialog));
 
     await tester.tap(find.text('X'));
-    await tester.pump(); // start animation
-    await tester.pump(const Duration(seconds: 1));
+    await tester.pumpAndSettle();
 
     expect(didPressOk, false);
     await tester.tap(find.text('OK'));
     expect(didPressOk, true);
   });
 
-  testWidgets('Dialog background color', (WidgetTester tester) async {
+  testWidgets('Dialog background color from theme', (WidgetTester tester) async {
```

Comment:
This title should be 'Dialog background color from AlertDialog'

Reply:
Done.